### ISSUE TITLE
Loading the External reference 

### ISSUE THREAD

**Issue by madhufeb** [faylang/fay#445]
Hi 
querying  about my project bootstrap file are not working when min.js placed above that bootstrap.min.js
is there and placement matters in js files

**Comment by bergmark** [comment]
Yes, the browser will parse and execute the first file before touching the next one.
